prime-router/build.gradle.kts

```diff
@@ -670,16 +651,16 @@ configurations {
 }
 
 dependencies {
-    jooqGenerator("org.postgresql:postgresql:42.4.0")
+    jooqGenerator("org.postgresql:postgresql:42.3.3")
```

Comment:
Several version rolebacks here re-introduce vulnerabilities.

Reply:
set to 42.4.1 as on master